src/aiortc/contrib/media.py

```diff
@@ -578,6 +578,10 @@ def _stop(self, proxy: RelayStreamTrack) -> None:
             # unregister proxy
             self.__log_debug("Stop proxy %s", id(proxy))
             self.__proxies[track].discard(proxy)
+            if len(self.__proxies[track]) == 0 and track in self.__tasks:
+                self.__tasks[track].cancel()
+                del self.__tasks[track]
```

Comment:
I'd rather have the un-registration happen in a single place. Couldn't we handle cancellation in `__run_track`?